### DISCUSSION TITLE
Other language support

### DISCUSSION THREAD

**Discussion by LafepuCollapse** [makeplane/plane#3979] [Q&A]
Hi
 Is it possible to support another language?

**Comment by moigamijunior** [comment]
I'ld like to work on it. Do you have news?

**Comment by ja49619** [comment]
UP. I can translate it into Russian

**Reply by imanmagomedov** [reply]
i can help you to translate it into Russian, bro!

**Comment by imanmagomedov** [comment]
i'm create a issue https://github.com/makeplane/plane/issues/5403

**Reply by ja49619** [reply]
there was no need to do this. The developer somewhere in the next topic wrote that they are developing translation platforms. just waiting

**Reply by imanmagomedov** [reply]
oh, didn't know... thx!

**Reply by imanmagomedov** [reply]
i'm close this issue
